examples/elastic-newstracker/top_linked_pages_write_rollup.juttle

```diff
@@ -1,8 +1,9 @@
-read elastic -from :2009-04-01: -to :2009-05-01: linkout != null
+read elastic -from :2009-04-01: -to :2009-05-01: linkout != null tag != 'rollup_linkout'
 | batch :1 day:
 | reduce count() by linkout
 | sort count -desc
 | head 100
 | put tag = 'rollup_linkout'
+| put time = time - :1 day:
```

Comment:
disturbing that we have to do that. There was some conversation about fixing the next-period-timestamped batching issue... not sure if the approach of having intervals would even address this. @demmer?